### PR TITLE
Test engine ci

### PR DESCRIPTION
**Pull request by Thoralf-M**

### CODE REVIEW

File: .github/actions/private-tangle/setup/action.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        # cd private_tangle
+        # # Use the output of https://api.github.com/repos/iotaledger/hornet/releases/latest once there's a 2.0 Hornet release.
+        # DOWNLOAD_URL=$(curl "https://api.github.com/repos/iotaledger/hornet/releases" | jq -r '.[0].assets[] | select(.name | contains("private_tangle")) | .browser_download_url')
+        # echo "Downloading private tangle from $DOWNLOAD_URL"
+        # curl -L -o private_tangle.tar.gz $DOWNLOAD_URL
+        # tar -xf private_tangle.tar.gz
+        # TODO: remove next lines when a working hornet release is published
+        git clone https://github.com/iotaledger/hornet.git
+        cd hornet/private_tangle
+        # Set minPoWScore = 1 since the default (0) doesn't work with wasm_miner.rs in iota.rs currently.
```
Comment (thibault-martinez) [inline .github/actions/private-tangle/setup/action.yml]: This has been fixed?

Comment (Thoralf-M) [reply .github/actions/private-tangle/setup/action.yml]: Yes, just copied the whole workflow from identity.rs, will update

File: tester/src/main.rs [hunk 2]
```diff
@@ -95,7 +95,7 @@ async fn main() -> Result<(), Error> {
     logger_init()?;
 
     let mut entries = Vec::new();
-    let mut dir = fs::read_dir("json").await?;
+    let mut dir = fs::read_dir("tester/json").await?;
```
Comment (thibault-martinez) [inline tester/src/main.rs]: I think I'd rather do a `cd` in the CI than change that here tbh 🤔

Comment (Thoralf-M) [reply tester/src/main.rs]: Where in the CI do the `cd`?

Comment (thibault-martinez) [reply tester/src/main.rs]: Before you clone hornet I think

Comment (thibault-martinez) [reply tester/src/main.rs]: What's the status with that ?

Comment (Thoralf-M) [reply tester/src/main.rs]: Didn't work

Comment (thibault-martinez) [reply tester/src/main.rs]: :carlos:

**Pull request merged**